### PR TITLE
Fix warnings & add bibliography to TOC

### PR DESCRIPTION
**Pull request by ElectroluxV2**
Fixed:
```
Package biblatex Warning: Using fall-back BibTeX(8) backend:
(biblatex)                functionality may be reduced/unavailable.
```
and:
```
Package biblatex Warning: 'babel/polyglossia' detected but 'csquotes' missing.
(biblatex)                Loading 'csquotes' recommended.
```

To include bibliographty in toc use `\printbibliography[heading=bibintoc]`

### CODE REVIEW

File: sprz.cls [hunk 1]
```diff
@@ -15,13 +15,13 @@
...
 \RequirePackage{tabularx}
-\RequirePackage[backend=bibtex,style=numeric,sorting=none]{biblatex}
+\RequirePackage[backend=biber,style=numeric,sorting=none]{biblatex}
 \RequirePackage{url}
 \RequirePackage{graphicx}
 \RequirePackage[pscoord]{eso-pic}% The zero point of the coordinate systemis the lower left corner of the page (the default).
 \RequirePackage{multirow}
 \RequirePackage{graphicx}
-
+\RequirePackage{csquotes}
```
Comment (pantadeusz) [inline sprz.cls]: Why introduce additional package that is not necessary?

File: engineer.tex [hunk 2]
```diff
@@ -150,9 +150,9 @@ \chapter{Wymagania wszelakie}
 \end{requirementstab}
 
 
-\printbibliography[title={Bibliografia}]
+\printbibliography[title={Bibliografia}, heading=bibintoc]
```
Comment (pantadeusz) [inline engineer.tex:154]: The bibliography in or not in table of contents is disputable. I will accept it for now.

File: sprz.cls [hunk 3]
```diff
@@ -15,13 +15,13 @@
 \RequirePackage[utf8]{inputenc}
 \RequirePackage{polski} % Włączenie obsługi polskich liter
 \RequirePackage{tabularx}
-\RequirePackage[backend=bibtex,style=numeric,sorting=none]{biblatex}
+\RequirePackage[backend=biber,style=numeric,sorting=none]{biblatex}
```
Comment (pantadeusz) [inline sprz.cls]: This is reasonable to change from bibtex to biber backend. ACK.

**Review by pantadeusz — CHANGES_REQUESTED** [review]
I would like to not introduce unnecessary packages. If someone don't want to paste citations in a text, then the csquotes package is not needed. Please remove it. Other changes are OK.

File: sprz.cls (hunk 1 above)
Comment (ElectroluxV2) [reply sprz.cls]: To remove warning from build logs. I went the other way and removed `biblatex` from template.

**Pull request merged**